last_gas/domain/schedulers/time_scheduler.py

```diff
@@ -61,5 +75,14 @@ async def last_gas(bot: Any, channel_id: str) -> None:
         "kwargs": {
             "channel_id": CHANNEL_IDS["geralt"],
         },
-    }
+    },
+    {
+        "timed_func": sad_mondays,
+        "day_of_week": 0,
+        "time_of_day": "09:00:00",
+        "args": [],
+        "kwargs": {
+            "channel_id": CHANNEL_IDS["geralt"],
+        },
+    },

```

Comment:
We could have this data model "Schedule" structured into a daclass model for developers to handle structured python objects instead of raw directories, which we know are prone to typos and other silly bugs.

Maybe something like this (?):

```python
@dataclass
class Schedule:
    timed_func: Callable[(Any, str), None]  # it is really `Any` here? I think we could use a better typing
    day_of_week: int
    time_of_day: str  # or maybe timedelta
    args: Sequence[Any]
    kwargs: Dict[str, Any]

```

Does it make sense?